src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -291,4 +305,12 @@ export default class TableComponent extends ComponentBase implements IComponent,
 	public get title() {
 		return this.getPropertyOrDefault<azdata.TableComponentProperties, string>((props) => props.title, '');
 	}
+
+	public get ariaRowCount(): number {
+		return this.getPropertyOrDefault<azdata.TableComponentProperties, number>((props) => props.ariaRowCount, -1);
```

Comment:
1) For my understanding - is it taking the value as it is -1? how does it remove "of 0 column" from speech?

2) Also, this will affect all tables so please take a quick look at some other table that they work... 
One easy one to check should be schema compare